@dao-auction/components/TokenWinningBid.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import { useDaoToken } from "@dao-auction/hooks/useDaoToken"
+import { useNounsProtocol } from '@dao-auction/hooks/useNounsProtocol'
+import { useActiveAuction } from '@dao-auction/hooks/useActiveAuction'
+import { ethers } from 'ethers'
+
+export default function TokenWinningBid({
+  tokenId,
+  daoAddress,
+}: {
+  daoAddress: string
+  tokenId: string
+  }) {
+  const {
+    auctionData
+  } = useActiveAuction(daoAddress)
+  
+  const { tokenData } = useDaoToken({
+    daoAddress: daoAddress,
+    tokenId: tokenId,
+  })
+  
+  const {
+    BuilderAuction
+  } = useNounsProtocol({daoAddress: daoAddress, auctionAddress: auctionData?.address})
+
+  const [winningBid, setWinningBid] = React.useState<string | undefined>('N/A')
+
+  React.useEffect(() => {
+    async function getBids() {
+      try {
+        if(tokenData?.mintInfo?.mintContext?.blockNumber) {
+          /** 
+           * https://docs.ethers.io/v5/api/contract/contract/#Contract-queryFilter
+           * Used to query the Auction events exposed below:
+           * https://github.com/ourzora/nouns-protocol/blob/main/src/auction/IAuction.sol#L16-L22
+           */
+          const bids = await BuilderAuction?.queryFilter(
```

Comment:
@0xTranqui / @salieflewis this ethers read call is soooo cool...

Reply:
Did you write `useNounsProtocol`, `useActiveAuction`?

Reply:
Yes.